### ISSUE TITLE
Pages on Angular.io fail to load w/ "Too Many Redirects" error

### ISSUE THREAD

**Issue by KyleWpppd** [angular/angular#43989]
### Description

Some angular.io documentation pages fail to load because of a `302 Redirect` loop. 

Examples:
- https://angular.io/guide/what-is-angular
- https://angular.io/cli/add
- https://angular.io/guide/forms
- https://angular.io/guide/attribute-directives
- https://angular.io/tutorial/toh-pt0

### What is the affected URL?

angular.io/cli/add

### Please provide the steps to reproduce the issue

# Browser via Direct URL

1. Open browser
2. Type `angular.io/cli/add` into address bar
3. Observe error page

# Browser via Clicking from Angular.io

1. Open browser
2. Visit angular.io
3. In the top menu, click on "Docs", be taken to documentation pages.
4. In the left documentation menu, click "Reference" then "CLI command reference" then "ng add"
5. Observe Angular.io's error page.

# CLI

1. Make a request via cURL `$ curl -i https://angular.io/cli/add`

```
HTTP/2 302
cache-control: no-cache
content-security-policy-report-only: require-trusted-types-for 'script'; trusted-types angular angular#bundler angular#unsafe-bypass aio#analytics google#safe; report-uri https://csp.withgoogle.com/csp/angular.io
link: </generated/navigation.json>;rel=preload;as=fetch,</generated/docs/index.json>;rel=preload;as=fetch
location: https://angular.io/cli/add
content-type: text/plain; charset=utf-8
accept-ranges: bytes
date: Sat, 30 Oct 2021 00:20:12 GMT
x-served-by: cache-pao17448-PAO
x-cache: MISS
x-cache-hits: 0
x-timer: S1635553213.513052,VS0,VE122
vary: x-fh-requested-host, accept-encoding
content-length: 41

Redirecting to https://angular.io/cli/add
```

2. Observe that the redirect is to the same page.

### Please provide the expected behavior vs the actual behavior you encountered

Expected
---------

The documentation renders

Actual
-------

The webpage renders with

> REQUEST FOR DOCUMENT FAILED
> We are unable to retrieve the "cli/add" page at this time.
> Please check your connection and try again later.

### Please provide a screenshot if possible

https://user-images.githubusercontent.com/597653/139513147-220c99ca-3eb9-42cb-9b15-b4d2e18f043f.mov

### Please provide the exception or error you saw

```true
Too many 302 redirects.

The attached video shows the 302s. 

Curl used to reproduce HTTP request & response.

`$ curl -i https://angular.io/cli/add`


HTTP/2 302
cache-control: no-cache
content-security-policy-report-only: require-trusted-types-for 'script'; trusted-types angular angular#bundler angular#unsafe-bypass aio#analytics google#safe; report-uri https://csp.withgoogle.com/csp/angular.io
link: </generated/navigation.json>;rel=preload;as=fetch,</generated/docs/index.json>;rel=preload;as=fetch
location: https://angular.io/cli/add
content-type: text/plain; charset=utf-8
accept-ranges: bytes
date: Sat, 30 Oct 2021 00:20:12 GMT
x-served-by: cache-pao17448-PAO
x-cache: MISS
x-cache-hits: 0
x-timer: S1635553213.513052,VS0,VE122
vary: x-fh-requested-host, accept-encoding
content-length: 41

Redirecting to https://angular.io/cli/add
```
```


### Is this a browser-specific issue? If so, please specify the device, browser, and version.

```true
No. I was able to reproduce in Chrome in macOS, iOS, and Linux. Additionally I was able to reproduce in Safari on macOS and iOS.
```

**Comment by pstricks-fans** [comment]
angular.io is offline now.

**Comment by gkalpak** [comment]
Thx for reporting this, @KyleWpppd. It should be fixed now.
@KyleWpppd, @pstricks-fans: Can you confirm that it works for you too?

**Comment by pstricks-fans** [comment]
Now it works very well.

**Comment by atscott** [comment]
Fixed by #43991
